src/components/RichSelect/__stories__/index.stories.tsx

```diff
@@ -334,6 +334,31 @@ LoadingExample.decorators = [
   },
 ]
 
+export const Description: Story = () => (
+  <RichSelect name="basic">
+    <RichSelect.Option
+      value="a"
+      inlineDescription="This is an average solution"
+    >
+      Option A
+    </RichSelect.Option>
+    <RichSelect.Option
+      value="b"
+      description="This is the best solution, friend advice"
```

Comment:
Should it be better to have `description={value: "blablabla" /* Or <div>blabla</div> */, inline: true}` ?
Or
`description="edededede" inlineDescription`

Like this description is a consistent prop which display a description 🤔

Reply:
@DorianMaliszewski That's a good one but in fact I tough that maybe one day we will have both inline and normal description to display at same time. If such case happen then my structure works better I think